KubeArmor/.goreleaser.yaml

```diff
@@ -10,6 +10,10 @@ builds:
       - arm64
     env:
       - CGO_ENABLED=0
+    ldflags:
+      - "-X main.BuildDate={{.Date}} -X main.GitCommit={{.Commit}} -X main.GitBranch={{.Branch}}  -X main.GitSummary={{.Summary}}"
```

Comment:
u can add the flags in different lines instead of writing all flags in single line